spacepy/pybats/gitm.py

```diff
@@ -8,8 +8,10 @@
 # Global imports:
 import numpy as np
 import datetime as dt
+from struct import unpack
 from spacepy.pybats import PbData
 from spacepy.datamodel import dmarray
+import glob
```

Comment:
We can probably take advantage of the time to clean up the imports, according to [PEP8](https://peps.python.org/pep-0008/#imports).

- Separate standard library / other libraries / internal
- We know it's global imports, no need to have the comment
- I like having them alphabetized within each section, easier to tell what's going on
- Personal preference: not a fan of from imports, but we should at least be _sort of_ consistent...the fact that glob and struct are different here makes me twitchy